build_tools/bazel/iree_e2e_generated_runner_test.bzl

```diff
@@ -44,23 +43,18 @@ def iree_e2e_runner_test(
             added automatically.
         test_runner: test runner program to run.
         timeout: timeout for the generated tests.
-        target_cpu_features: target CPU features. Only for llvm-cpu backend.
         **kwargs: any additional attributes to pass to the underlying tests and
             test suite.
     """
 
-    if target_cpu_features:
-        fail("target_cpu_features must currently be empty")
-
     iree_bytecode_module(
         name = name + "_%s_module" % test_type,
         module_name = tests_vmfb,
         src = tests_src,
         flags = [
             "--iree-hal-target-backends=%s" % target_backend,
-        ] + ([
-            "--iree-llvmcpu-target-cpu-features=%s" % target_cpu_features,
-        ] if target_cpu_features else []) + compiler_flags,
+            "--iree-llvmcpu-target-cpu=generic",
+        ] + compiler_flags,
```

Comment:
Same comment here about implicit flags in the helper function vs explicit flags at the call sites

Reply:
same.